content/en/observability_pipelines/setup/splunk.md

```diff
@@ -20,18 +20,16 @@ further_reading:
 <div class="alert alert-warning">Observability Pipelines is not available on the US1-FED Datadog site.</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">Observability Pipelines only supports Splunk's HTTP Event Collector (HEC) protocol.</div>
-
 ## Overview
 
 The [Observability Pipelines Worker][1] can collect, process, and route logs and metrics from any source to any destination. Using Datadog, you can build and manage all of your Observability Pipelines Worker deployments at scale.
 
-This guide walks you through deploying the Worker in your common tools cluster and configuring Splunk to send logs through the Worker, to dual-write to Datadog.
+This guide walks you through deploying the Worker in your common tools cluster and configuring your Splunk Heavy/Universal Forwarders to send logs through the Worker, to dual-write to Datadog.
```

Comment:
Since we also include instructions for HEC should we also mention it here?

```suggestion
This guide walks you through deploying the Worker in your common tools cluster and configuring your Splunk Heavy/Universal Forwarders and HEC to send logs through the Worker, to dual-write to Datadog.
```